wagtail/contrib/modeladmin/options.py

```diff
@@ -243,13 +243,16 @@ def get_ordering(self, request):
 
     def get_queryset(self, request):
         """
-        Returns a QuerySet of all model instances that can be edited by the
-        admin site.
+        Returns a QuerySet of all model instances that can be explored by the
+        current user.
         """
         qs = self.model._default_manager.get_queryset()
+        qs = filter_explorable_pages(qs, request.user, include_ancestors=False)
```

Comment:
"Explorable" isn't a specific permission: it's the equivalent of "the user has at least one of the following perms: Add, Edit, Publish, or Lock". Only "Choosable" is a specific permission.

The reason I put the explorability restriction on the ModelAdmin listing was to prevent non-superusers from seeing Pages that belong to other Sites, which is exactly what the explorability filter on the Explorer is all about.